### ISSUE TITLE
Converge fails due to missing dependency alfresco-rm-core-repo-2.4.amp

### ISSUE THREAD

**Issue by smford22** [Alfresco/chef-alfresco#82]
The README states that you should be able to converge this cookbook with `kitchen converge community` but converge fails due to the file `alfresco-rm-core-repo-2.4.amp` missing on the Nexus repo.  There does not appear to be an `alfresco-rm-core` tree on the Nexus server at all.

```
================================================================================
           Error executing action `put` on resource 'maven[alfresco-rm-core-repo]'
           ================================================================================

           Mixlib::ShellOut::ShellCommandFailed
           ------------------------------------
           Expected process to exit with [0], but received '1'
           ---- Begin output of mvn org.apache.maven.plugins:maven-dependency-plugin:2.4:get -DgroupId=org.alfresco -DartifactId=alfresco-rm-core-repo -Dversion=2.4 -Dpackaging=amp  -Ddest=/chef_maven_lwrp20160719-2982-cl0866/alfresco-rm-core-repo.amp -DremoteRepositories=public::::https://artifacts.alfresco.com/nexus/content/groups/public -Dtransitive=false ----
           STDOUT: [INFO] Scanning for projects...
           [INFO]
           [INFO] ------------------------------------------------------------------------
           [INFO] Building Maven Stub Project (No POM) 1
           [INFO] ------------------------------------------------------------------------
           [INFO]
           [INFO] --- maven-dependency-plugin:2.4:get (default-cli) @ standalone-pom ---
           Downloading: https://artifacts.alfresco.com/nexus/content/groups/public/org/alfresco/alfresco-rm-core-repo/2.4/alfresco-rm-core-repo-2.4.amp
[INFO] ------------------------------------------------------------------------
           [INFO] BUILD FAILURE
           [INFO] ------------------------------------------------------------------------
           [INFO] Total time: 2.649s
           [INFO] Finished at: Tue Jul 19 02:22:06 UTC 2016
           [INFO] Final Memory: 13M/106M
           [INFO] ------------------------------------------------------------------------
           [ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.4:get (default-cli) on project standalone-pom: Couldn't download artifact: Failure to find org.alfresco:alfresco-rm-core-repo:amp:2.4 in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced
```

**Comment by enzor** [comment]
hi there Scott.
We are actively working on debugging this issue, will get you back ASAP! :)
@marcomancuso  can you give us a hand with this?

**Comment by enzor** [comment]
@marcomancuso delivered a solution in the pull request #83 . I am currently testing that, will get back to you as soon as i can.

**Comment by enzor** [comment]
blocked by #85  .

**Comment by enzor** [comment]
fixed by #83 .
Please try master again, and let me know if anything fails. Otherwise i will close the issue
